src/hostnet/hostnet_icmp.ml

```diff
@@ -145,9 +145,9 @@ module Make
             let flow = Hashtbl.find t.phys_to_flow (src, id) in
             let id' = snd flow.virt in
             (* Rewrite the id in the Echo response *)
-            Icmpv4_wire.set_icmpv4_id raw id';
-            Icmpv4_wire.set_icmpv4_csum raw 0;
-            Icmpv4_wire.set_icmpv4_csum raw (Tcpip_checksum.ones_complement raw);
+            Cstruct.BE.set_uint16 raw 5 id'; (* TODO: Upstream *)
```

Comment:
nice catch, please upstream the missing bindings (from when I removed ppx_cstruct)